module-3/app/service/client/mysfitsTableClient.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// Info is for logging
+var Info *log.Logger
+
+// Init initializes the logger
+func Init(
+	infoHandle io.Writer) {
+	Info = log.New(infoHandle,
+		"INFO: ",
+		log.Ldate|log.Ltime|log.Lshortfile)
+}
+
+// Item is a value returned by query
+type Item struct {
```

Comment:
Should this be named "Mysfit" instead of Item?

Reply:
Done